### PR TITLE
Rename `more-dropdown` to `more-dropdown-links`

### PR DESCRIPTION
**Pull request by yakov116**
Migration will happen in #4568, after the feature is renamed.

### CODE REVIEW

**Review by fregante — APPROVED** [review]

**Comment by fregante** [comment]
Unchecked but I assume it matches the previous code. Merge at will

File: readme.md [hunk 1]
```diff
@@ -134,6 +134,7 @@ Thanks for contributing! 🦋🙌
 - [](# "hide-watch-and-fork-count") [Hides watcher counter and on smaller screens the fork counter too.](https://user-images.githubusercontent.com/1402241/53681077-f3328b80-3d1e-11e9-9e29-2cb017141769.png)
 - [](# "sort-milestones-by-closest-due-date") Changes the default sort order of milestones `Closest due date`.
 - [](# "more-dropdown") [Adds useful links to the repository navigation dropdown and moves the "Security" and "Insights" tabs to it as well.](https://user-images.githubusercontent.com/16872793/124681432-856e6f80-de96-11eb-89c9-6d78e8ae4329.png)
```
Comment (FloEdelmann) [inline readme.md:136]: This line should be dropped, right?

File: source/features/remove-unused-repo-tabs.tsx [hunk 2]
```diff
@@ -83,7 +83,20 @@ async function initProjects(): Promise<void | false> {
 	projectsTab!.remove();
 }
 
+async function init(): Promise<void> {
+	// The user may have disabled `more-dropdown` so un-hide it
```
Comment (FloEdelmann) [inline source/features/remove-unused-repo-tabs.tsx:87]:
```suggestion
	// The user may have disabled `more-dropdown-links` so un-hide it
```

**Review by FloEdelmann — CHANGES_REQUESTED** [review]

**Comment by kdcloverkid** [comment]
more-dropdown-links

**Pull request merged**